frontend/benefit/handler/src/components/messenger/useMessenger.ts

```diff
@@ -35,6 +37,14 @@ const useMessenger = (): ExtendedComponentProps => {
     MESSAGE_URLS.NOTES
   );
 
+  const doPolling = React.useCallback((): void => {
```

Comment:
I dont think you need polling, this is what react-query does automatically for you. I havent used refetch myself but I think you can just set  `refetchInterval` in the useQuery options inside useMessagesQuery: https://react-query.tanstack.com/reference/useQuery

Reply:
cool, i ll test!

Reply:
fixed